packages/yuudachi/locales/en-GB/translation.json

```diff
@@ -124,25 +125,59 @@
 			},
 			"anti_raid_nuke": {
 				"errors": {
+					"invalid_file_type": "Invalid file type provided, must be a `.txt` or `.json` file.",
+					"invalid_avatar": "Invalid avatar format, must be an raw hash, a user id, a URL or `none`.",
 					"no_hits": "The Anti-Raid-Nuke will not hit anyone, you might want to re-adjust the parameters and try again!",
-					"parameters": {
-						"heading": "Parameters:",
-						"current_time": "• Current time: {{- now}}",
-						"join_after": "• Join date after: {{- join}}",
-						"created_after": "• Account creation after: {{- age}}",
-						"pattern": "• Username matching pattern:",
-						"avatar": "• Avatar hash (user or member) matching: {{- avatar}}",
-						"days_one": "• Deleting messages sent in the last {{count}} day",
-						"days_other": "• Deleting messages sent in the last {{count}} days",
-						"days_none": "• Not deleting any messages"
+					"no_hits_file": "The Anti-Raid-Nuke will not hit anyone, content you provided did not contain any valid ids.",
+					"no_ids": "This file does not contain any ids.",
+					"duration_format": "Wrong format for the argument {{ arg }} argument",
+					"pattern_format": "Invalid pattern provided",
+					"result": {
+						"ban_failed": "There was an error trying to ban: {{- error}}",
+						"case_failed": "There was an error trying to create a case (ban succeeded): {{- error}}",
+						"reject_perms": "The member has elevated permissions.",
+						"reject_unbanable": "The member is not bannable by me.",
+						"reject_bot": "The member is a bot.",
+						"reject_self": "The member is you.",
+						"reject_protected": "The member has a role ignored by the Anti-Raid-Nuke.",
+						"reject_user": "The user is not in the guild."
+					}
+				},
+				"parameters": {
+					"heading": "Parameters:",
+					"current_time": "• Current time: {{- now}}",
+					"dry_run": "• Operating in **dry-run** mode, no members will be banned!",
+					"join_after": "• **Join:**\n> From: {{- joinFrom}}\n> To: {{- joinTo}}",
+					"created_after": "• **Account creation:** \n> From: {{- ageFrom}}\n> To: {{- ageTo}}",
+					"pattern": "• Username matching pattern:",
+					"avatar": "• Avatar hash (user or member) matching: {{- avatar}}",
+					"zalgo": "• Filtering usernames with Zalgo",
+					"confusables": "• Filtering usernames with Confusables",
+					"days": "• Deleting messages sent in the last {{count}} day",
+					"days_plural": "• Deleting messages sent in the last {{count}} days",
+					"days_none": "• Not deleting any messages",
+					"file": "• File: {{- file}}",
+					"users": "• From all of the ids, {{- users}} aren't members of this guild"
```

Comment:
From is wrong here, should be something like "out of"
from denotes the beginning of a range